Sources/Apodini/Semantic Model Builder/REST/RESTEndpointHandler.swift

```diff
@@ -0,0 +1,71 @@
+//
+// Created by Andi on 30.12.20.
+//
+
+import Foundation
+@_implementationOnly import Vapor
+import protocol FluentKit.Database
+
+extension Vapor.Request: ExporterRequest, WithEventLoop, WithDatabase {
+    var database: () -> Database {{
+        self.db
+    }}
```

Comment:
Since #84 is merged, I think we don't need this property anymore. Other than that it looks good 👍

Reply:
Ah didn't see it got merged. Will update this branch accordingly and ping you then 👍

Reply:
Rebased onto the latest develop state and removed the database workaround via https://github.com/Apodini/Apodini/pull/86/commits/84389ca086fd57e63a0e54443decbc3e850b53bf.

I hope its fine that I just included that with this PR.